packages/block-editor/src/components/index.js

```diff
@@ -47,6 +47,7 @@ export { default as __experimentalDateFormatPicker } from './date-format-picker'
 export { default as __experimentalDuotoneControl } from './duotone-control';
 export { default as __experimentalFontAppearanceControl } from './font-appearance-control';
 export { default as __experimentalFontFamilyControl } from './font-family';
+export { default as HTMLElementSelectControl } from './html-element-select-control';
```

Comment:
Suggestion (blocking): Almost all new components should start as private APIs. They can be stabilized and made public later as needed.

You can find examples of private components/APIs in `packages/block-editor/src/private-apis.js`.